src/compiler/diagnosticMessages.json

```diff
@@ -1759,6 +1759,14 @@
         "category": "Error",
         "code": 2658
     },
+    "'super' in members of object literal expressions is only allowed when option 'target' is 'ES2015' or higher.": {

```

Comment:
`"The 'super' keyword is only allowed in an object literal expression when targetting 'ES2015' or higher."`